lib/screens/icon_detail_widget.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:mixeditems/models/icon_item.dart';
+import 'package:mixeditems/repository/widget_placeholder.dart';
+
+class IconDetailWidget extends StatelessWidget {
+  const IconDetailWidget({Key? key, required this.iconItem}) : super(key: key);
+
+  final IconItem iconItem;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('App Info'),
+      ),
+      body: _buildListView(context),
+    );
+  }
+
+  Widget _buildListView(BuildContext context) {
+    WidgetPlaceHolder _widgetPlaceHolder = WidgetPlaceHolder();
```

Comment:
This should be a property of the class, instead of being newly created everytime this function is called

Reply:
I see I will call them in the widget class.